draft-ietf-httpbis-http2bis.xml

```diff
@@ -5057,15 +5057,11 @@ cookie: e=f
         <li>TLS_PSK_WITH_AES_128_CCM_8</li>
         <li>TLS_PSK_WITH_AES_256_CCM_8</li>
       </ul>
-      <aside>
-        <t>Note:
-          This list was assembled from the set of registered TLS cipher suites at the time of
-          writing.  This list includes those cipher suites that do not offer an ephemeral key
-          exchange and those that are based on the TLS null, stream, or block cipher type (as
-          defined in <xref target="TLS12" section="6.2.3"/>).  Additional cipher suites with
-          these properties could be defined; these would not be explicitly prohibited.
-        </t>
-      </aside>
+      <aside><t>Note: This list was assembled from the set of registered TLS cipher suites when
+      <xref target="RFC7540"/> was developed.  This list includes those cipher suites that do not
```

Comment:
Presumably this was meant to be `excludes`, not `includes`, and we just missed it?

Reply:
Includes is correct.  The negation comes later.

Reply:
It comes earlier, as it turns out, but great.